caas/mixins.spec

```diff
@@ -54,7 +54,7 @@ power: true(power_throttle=true)
 debug-usb-config: true(source_dev=dvcith-0-msc0)
 intel_prop: true
 trusty: true(ref_target=celadon_64)
-memtrack: false
+memtrack: true
```

Comment:
does it mean memtrack for all CiV target?

Reply:
I think turn on the memtrack here is enable the AIDL hal.
The aidl service is enabled for all civ target.
Also it need kernel have related function and related so to load the kernel files.
We use the memtrack.celadon.so.
The memtrack.celadon.so is implemented before by Shi, qiming, it is deal with the i915 driver.